app/src/main/java/org/mifos/mobile/ui/fragments/SavingAccountsTransactionFragment.kt

```diff
@@ -298,84 +299,70 @@ class SavingAccountsTransactionFragment : BaseFragment() {
     /**
      * Shows a filter dialog
      */
+     private lateinit var filterBinding: LayoutFilterDialogBinding
```

Comment:
make this global and change the name of variable to layoutFilterDialogBinding

Reply:
Okay...needed to change the first letter to small